File: datasets/IR-Plag/convertai-original-typescript-rest-java/case-03/plagiarized/L1/05/percabangan.java
package IR;

import java.util.Scanner;

public class percabangan {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter weight in pounds: ");
        double weight = input.nextDouble();//inputan weight
        System.out.print("Enter feet: ");
        double feet = input.nextDouble();//inputan feet
        System.out.print("Enter inches: ");
        double inches = input.nextDouble();//inputan inches
        double height = feet * 12 + inches;// menghitung tinggi       
        double bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254)); //menghitung BMI
        //menampilkan hasil
        System.out.println("BMI is " + bmi);
        if (bmi < 18.5) {
            System.out.println("Underweight");
        } else if (bmi < 25) {
            System.out.println("Normal");
        } else if (bmi < 30) {
            System.out.println("Overweight");
        } else {
            System.out.println("Obese");
        }
    }
}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-03/plagiarized/L1/05/percabangan.kt
package IR

import java.util.*

object percabangan {
    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)
        print("Enter weight in pounds: ")
        val weight = input.nextDouble() //inputan weight
        print("Enter feet: ")
        val feet = input.nextDouble() //inputan feet
        print("Enter inches: ")
        val inches = input.nextDouble() //inputan inches
        val height = feet * 12 + inches // menghitung tinggi       
        val bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254)) //menghitung BMI
        //menampilkan hasil
        println("BMI is $bmi")
        if (bmi < 18.5) {
            println("Underweight")
        } else if (bmi < 25) {
            println("Normal")
        } else if (bmi < 30) {
            println("Overweight")
        } else {
            println("Obese")
        }
    }
}
